homer.root/usr/share/clearwater/cassandra-schemas/homer.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
+
+cassandra_installed=$(dpkg-query -W -f='${Status}\n' cassandra | grep -q "install ok installed")
```

Comment:
Worth pulling this out into a common function (given that there's other schemas to update as well)?